frontend/packages/knative-plugin/src/topology/components/groups/KnativeService.tsx

```diff
@@ -9,7 +9,7 @@ import {
 } from '@console/topology';
 import { modelFor, referenceFor } from '@console/internal/module/k8s';
 import { useAccessReview } from '@console/internal/components/utils';
-import { getTopologyResourceObject } from '../../topology-utils';
+import { getTopologyResourceObject } from '@console/dev-console/src/components/topology/topology-utils';
```

Comment:
Definitely need a plugin infrastructure for utilities -- I do not currently see one in place.

cc @christianvogt

Reply:
Correct. That is the longer term goal. This PR is a stepping stone towards that.